src/applications/disability-benefits/all-claims/pages/additionalBehaviorChanges.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { PtsdNameTitle } from '../content/ptsdClassification';
+
+const additionalDescriptionChanges = (
+  <div>
+    <h5>Changes in Behavior or Activities: Additional Information</h5>
+    <p>
+      Please provide any additional information about any behavior changes or
+      actions you took as a result of the event(s).
+    </p>
+  </div>
+);
+
+export const uiSchema = {
+  'ui:title': ({ formData }) => (
+    <PtsdNameTitle formData={formData} formType="781a" />
+  ),
+  'ui:description': additionalDescriptionChanges,
+  additionalChanges: {
+    'ui:title': ' ',
+    'ui:widget': 'textarea',
+    'ui:options': {
+      rows: 5,
+      maxLength: 32000,
+    },
+  },
+};
+
+export const schema = {
+  type: 'object',
+  properties: {
+    additionalChanges: {
```

Comment:
I think this field needs to be added to `all-claims/config/schema.js`?

Reply:
@jbalboni: I've addd to all-claims/config/shema.js
Thank you!